<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>ExtraDirPanel</class>
  <widget class="QDialog" name="ExtraDirPanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>446</width>
        <height>362</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Extra Dir</string>
    </property>
    <property name="windowIcon">
      <iconset resource="resources.qrc" >
      <normaloff>:/images/images/LDViewIcon16.png</normaloff>:/images/images/LDViewIcon16.png</iconset>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QListWidget" name="ExtraDirListView">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>3</hsizetype>
                  <vsizetype>3</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
            </widget>
          </item>
          <item>
            <layout class="QVBoxLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>6</number>
              </property>
              <item>
                <widget class="QPushButton" name="addExtraDirButton">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>0</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Add</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="delExtraDirButton">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>5</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Delete</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="upExtraDirButton">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>5</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Up</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QPushButton" name="downExtraDirButton">
                  <property name="sizePolicy">
                    <sizepolicy>
                      <hsizetype>5</hsizetype>
                      <vsizetype>0</vsizetype>
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                    </sizepolicy>
                  </property>
                  <property name="text">
                    <string>Down</string>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="Spacer3">
                  <property name="sizeHint">
                    <size>
                      <width>20</width>
                      <height>20</height>
                    </size>
                  </property>
                  <property name="sizeType">
                    <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="orientation">
                    <enum>Qt::Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <item>
            <spacer name="Spacer13">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="okButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>OK</string>
              </property>
              <property name="shortcut">
                <string/>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancelButton">
              <property name="sizePolicy">
                <sizepolicy>
                  <hsizetype>0</hsizetype>
                  <vsizetype>0</vsizetype>
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Cancel</string>
              </property>
              <property name="shortcut">
                <string>Esc</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <tabstops>
    <tabstop>addExtraDirButton</tabstop>
    <tabstop>delExtraDirButton</tabstop>
    <tabstop>upExtraDirButton</tabstop>
    <tabstop>downExtraDirButton</tabstop>
    <tabstop>okButton</tabstop>
    <tabstop>cancelButton</tabstop>
  </tabstops>
  <pixmapfunction></pixmapfunction>
  <resources>
   <include location="resources.qrc" />
  </resources>
</ui>
